### ISSUE TITLE
Project can't build successfully with Gradle.

### ISSUE THREAD

**Issue by xiangwangcheng** [hivemq/hivemq-community-edition#109]
## Expected behavior
project builds successfully.
## Actual behavior
![image](https://user-images.githubusercontent.com/8049014/68863158-bfbb5400-0729-11ea-93aa-3d68b703b6fb.png)

## Details
- Affected HiveMQ CE version(s): 
- Used JVM version:

**Comment by SgtSilvio** [comment]
Hi @xiangwangcheng 
It seems like you can not download gradle plugins from plugins.gradle.org.
Can you please retry and comment if the problem still persists.

**Comment by xiangwangcheng** [comment]
@SgtSilvio  Thanks for the reply. Now I encounter another problem when build the project. 
![image](https://user-images.githubusercontent.com/8049014/69638185-2d09a600-1095-11ea-8ea4-677b3532ef30.png)

I am totally a beginner to gradle..

**Comment by SgtSilvio** [comment]
@xiangwangcheng You are using a wrong gradle version.
The project includes a gradle wrapper script that uses the right gradle version.
Also there is no `compile` task, use `packaging` instead.
=> Just call `./gradlew clean packaging`

**Comment by xiangwangcheng** [comment]
@SgtSilvio Thank you! It works.

**Comment by xiangwangcheng** [comment]
@SgtSilvio  Sorry to bother you again, but  I encountered this error when I tried to open it in IDEA.
![image](https://user-images.githubusercontent.com/8049014/69689372-bc4da280-1103-11ea-8442-bd9cb6d76d2a.png)

and I have another Gradle of version 6.0.1. I don't know whether there is some connections or not.
![image](https://user-images.githubusercontent.com/8049014/69689626-69281f80-1104-11ea-92b8-761494d90013.png)
